packages/react-native/Libraries/Components/View/ReactNativeStyleAttributes.js

```diff
@@ -111,6 +111,7 @@ const ReactNativeStyleAttributes: {[string]: AnyAttributeType, ...} = {
    * Transform
    */
   transform: {process: processTransform},
+  transformOrigin: true,
```

Comment:
Use this to pre-process it on the JS side so you can share it with the Android side - https://gist.github.com/jacobp100/86bc3fa863e41f42ca091386f6252f29 - I maintain [css-to-react-native](https://github.com/styled-components/css-to-react-native) so you can trust me 🤣

It gives you an array with 3 components corresponding to `x`, `y`, and `z`. Gives either a number ([json isKindOfClass:NSNumber.class]), or a string with a `%` at the end (re-use your existing logic here)